_maps/map_files/Pahrump/Pahrump.dmm

```diff
@@ -4500,7 +4500,7 @@
 "hru" = (/obj/structure/grille,/obj/structure/window/reinforced/fulltile,/obj/machinery/door/poddoor/preopen{id = "outerkebab_window"},/turf/open/floor/f13{tag = "icon-floorrusty"; icon_state = "floorrusty"},/area/f13/city)
 "huD" = (/obj/effect/spawner/lootdrop/f13/foodspawner,/turf/open/indestructible/ground/outside/desert,/area/f13/wasteland)
 "hvP" = (/obj/machinery/mineral/processing_unit_console,/turf/closed/wall/r_wall/rust,/area/f13/bunker)
-"hwH" = (/obj/effect/landmark/start/f13/vetdecanus,/turf/open/floor/plasteel/f13{icon_state = "plating"},/area/f13/legion)
+"hwH" = (/obj/effect/landmark/start/f13/decanus,/turf/open/floor/plasteel/f13{icon_state = "plating"},/area/f13/legion)

```

Comment:
Also your travis failed because the landmarks for the removed jobs is still in the .dmm